src/backend/distributed/deparser/ruleutils_15.c

```diff
@@ -430,12 +430,6 @@ static void get_coercion_expr(Node *arg, deparse_context *context,
 				  Node *parentNode);
 static void get_const_expr(Const *constval, deparse_context *context,
 			   int showtype);
-static void get_json_constructor(JsonConstructorExpr *ctor,
```

Comment:
can you explain how you reverted the changes in this file? Mostly for future reference

Reply:
Okay, putting that in the commit description and PR description as well